airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -186,6 +186,42 @@ def post(
         response = self._session.post(self.BASE_URL + url, params=params, json=data)
         return self._parse_and_handle_errors(response), response
 
+    def get_custom_object_schemas(self) -> Mapping[str, Any]:
+        data, response = self.get("/crm/v3/schemas", {})
+        schemas = {}
+        if response.ok and "results" in data:
+            for raw_schema in data["results"]:
+                schema = self.generate_schema(raw_schema=raw_schema)
+                schemas[raw_schema["name"]] = schema
+        return schemas
+
+    def generate_schema(self, raw_schema: Mapping[str, Any]) -> Mapping[str, Any]:
+        properties = {
+            "id": {"type": ["null", "string"]},
+            "createdAt": {"type": ["null", "string"], "format": "date-time"},
+            "updatedAt": {"type": ["null", "string"], "format": "date-time"},
+            "archived": {"type": ["null", "boolean"]},
+            "properties": {}
```

Comment:
It misses the `type` attribute. Some destination connectors can crash because of that.

Reply:
Good catch. Added that in [eb427b1](https://github.com/airbytehq/airbyte/pull/16032/commits/eb427b19ca401c76cabc66290bb21bf23b609f51).